build/LibraryVersions.g.cs

```diff
@@ -19,149 +19,151 @@ public static class LibraryVersion
             "TestApplication.Azure",
             new List<PackageBuildInfo>
             {
-                new("12.13.0"),
-                new("12.19.1"),
+                new("12.13.0", Array.Empty<string>()),
+                new("12.19.1", Array.Empty<string>()),
             }
         },
         {
             "TestApplication.Elasticsearch",
             new List<PackageBuildInfo>
             {
-                new("8.0.0"),
-                new("8.10.0"),
-                new("8.12.0"),
+                new("8.0.0", Array.Empty<string>()),
+                new("8.10.0", Array.Empty<string>()),
+                new("8.12.0", Array.Empty<string>()),
             }
         },
         {
             "TestApplication.EntityFrameworkCore",
             new List<PackageBuildInfo>
             {
-                new("6.0.12"),
-                new("7.0.13"),
+                new("6.0.27", Array.Empty<string>()),
+                new("7.0.16", Array.Empty<string>()),
+                new("8.0.2", new string[] {"net8.0"}),
             }
         },
         {
             "TestApplication.EntityFrameworkCore.Pomelo.MySql",
             new List<PackageBuildInfo>
             {
-                new("6.0.2"),
-                new("7.0.0"),
+                new("6.0.2", Array.Empty<string>()),
+                new("7.0.0", Array.Empty<string>()),
+                new("8.0.0", new string[] {"net8.0"}),
```

Comment:
Wondering if it is supported already. 

```suggestion
                new("8.0.0", ["net8.0"]),
```

Reply:
.NET 7 on centos, it can be hard, I would keep as is for now

Reply:
I mean the new shorter array literals with latest C# version 😄

Reply:
We do not have the latest version on Centos builds :] There is only .NET7